lib/member_page.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+require 'scraped'
+
+class MemberPage < Scraped::HTML
+  field :birth_date do
+    date_from(
+      record_for('Datël')
+    ).to_s
+  end
+
+  field :birth_place do
+    record_for('Vendlindja:')
+  end
+
+  field :area do
+    record_for('Qarku:')
+  end
+
+  # TODO: This isn't so easy to extract as the text varies
+  # field :commissions do
+  #   puts line_for('Komisioni').split(/(Komisioni)/).drop(1).join('').tidy
+  # end
+
+  private
+
+  def record_for(str)
+    table.xpath(".//span[contains(., '#{str}')]")
+         .text
+         .split(':')
+         .drop(1)
+         .last
+         .to_s
+         .chomp('.')
+         .tidy
```

Comment:
`.chomp('.').tidy` is in a slightly unexpected order, as it only removes the '.' if it's exactly the last character (i.e. it doesn't if the string ends in '. ') 

Is that deliberate behaviour?